lib/rademade_admin/model/adapter/query/sql.rb

```diff
@@ -30,16 +30,26 @@ def collect_where_conditions(where_conditions)
           end
 
           def build_where_condition(field: nil, value: nil)
-            field = "`#{field}`"
             if value.is_a? Regexp
-              ["LOWER(#{field}) REGEXP ?", [value.source]]
+              ["LOWER(#{field}) #{regexp_operator} ?", [value.source]]
             elsif value.is_a? Array
               ["#{field} IN (?)", [value]]
             else
               ["#{field} = ?", [value]]
             end
           end
 
+          def is_postgresql?
+            ::ActiveRecord::Base.connection_config[:adapter] == 'postgresql'
+          end
+
+          def regexp_operator
+            if is_postgresql?
+              "~"
+            else
+              "REGEXP"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.